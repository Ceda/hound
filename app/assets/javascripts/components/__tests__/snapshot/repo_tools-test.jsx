import RepoTools from '../../repo_tools.js';

it('renders appropriately without Show Private button (not syncing)', () => {
  const has_private_access = true;

  const onSearchInput = jest.genMockFunction();
  const onRefreshClicked = jest.genMockFunction();
  const onPrivateClicked = jest.genMockFunction();

  const component = renderer.create(
    <RepoTools
      showPrivateButton={!has_private_access}
      onSearchInput={(event) => onSearchInput}
      onRefreshClicked={(event) => onRefreshClicked}
      onPrivateClicked={(event) => onPrivateClicked}
      isSyncing={false}
    />

  );
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
