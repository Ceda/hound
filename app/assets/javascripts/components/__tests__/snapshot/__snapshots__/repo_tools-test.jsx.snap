exports[`test renders appropriately without Show Private button (not syncing) 1`] = `
<div
  className="repo-tools">
  <RepoToolsSearch
    onSearchInput={[Function]} />
  <RepoToolsRefresh
    isSyncing={false}
    onRefreshClicked={[Function]} />
</div>
`;
